utils/v2tofhirr4/mappings.bal

```diff
@@ -24,13 +25,12 @@ import ballerinax/health.hl7v26;
 import ballerinax/health.hl7v27;
 import ballerinax/health.hl7v28;
 import ballerinax/health.fhir.r4 as r4;
-import ballerina/log;
 
 # Parse a string to an HL7 message.
 #
 # + msg - HL7 message as a string
 # + return - hl7v2:Message
-public function stringToHl7(string msg) returns hl7:Message|error {
+public isolated function stringToHl7(string msg) returns hl7:Message|error {
```

Comment:
Can be an expression-bodied function?

Reply:
Fixed with 94531a35d0d70f447f75dc76836ac00027c54416